### PR TITLE
[15.0][MIG]account_due_list_aging_comment:Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: account_due_list_aging_comment/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+To use this module, you need to go to:
+
+Accounting -> Adviser -> Payments and due list
```
Comment (RodrigoBM) [inline account_due_list_aging_comment/readme/USAGE.rst]:
```suggestion
Accounting -> Payments and due list
```

**Comment by RodrigoBM** [comment]
this pr is still in wip? @bizzappdev

**Comment by bizzappdev** [comment]
MR Is ready for review.

**Review by RodrigoBM — APPROVED** [review]

**Comment by agustin-eticco** [comment]
Why this MR is not merged if has been aproved by @RodrigoBM?

**Pull request closed without merging**